src/screens/ContactScreen/index.js

```diff
@@ -262,9 +282,8 @@ const styles = StyleSheet.create({
   containerStyle: {
     marginHorizontal: 16
   },
-  subtitleStyle: (selectedUsers) => ({
-    color: selectedUsers.length > 0 ? COLORS.anon_primary : COLORS.gray4,
-    marginEnd: 8
+  subtitleStyle: (sourceScreen) => ({
+    color: sourceScreen === ANONYMOUS ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the subtitle is now dynamic based on whether the source screen is `ANONYMOUS` or not. This is a good change for improving the user interface and making it more intuitive.


<!-- This is an auto-generated comment by OSS CodeRabbit -->